pkg/service/main.go

```diff
@@ -25,9 +26,17 @@ import (
 	"github.com/gorilla/mux"
 )
 
+var serviceKeyOverride string
+
 func Main(serviceName string, serviceVersion string, proto interface{}, ctx context.Context, cancel context.CancelFunc, router *mux.Router, readyStream chan<- bool) {
 	startupTimer := startup.NewStartUpTimer(common.BootRetrySecondsDefault, common.BootTimeoutSecondsDefault)
 
+	sdkFlags := flags.New()
```

Comment:
Need to add the usesage for you addition command-line option. Here is a sample from App Funcs SDK:

```
	additionalUsage :=
		"    -s/--skipVersionCheck           Indicates the service should skip the Core Service's version compatibility check.\n" +
			"    -sk/--serviceKey                Overrides the service key to be used with Registry and/or Configuration Providers\n" +
			"                                    If the name provided contains the text `<profile>`, this text will be replaced with\n" +
			"                                    the name of the profile used."

	sdkFlags := flags.NewWithUsage(additionalUsage)

```